tests/fiaas_deploy_daemon/e2e_expected/v3full-deployment.yml

```diff
@@ -229,6 +241,16 @@ spec:
           name: fiaas-secrets-init-container
           optional: true
         name: fiaas-secrets-init-container-config
+      - configMap:
+          defaultMode: 420
+          name: test1
+          optional: true
+        name: test1-config
+      - configMap:
+          defaultMode: 420
+          name: test2
+          optional: true
+      - name: test2-config
```

Comment:
A `- ` too much on this line I think. Should solve the remaining two tests I guess.